buildspec.yml

```diff
@@ -1,29 +0,0 @@
-version: 0.2
```

Comment:
Did we remove the old cloj dependencies (for aws) from the project.clj file in a previous commit? I can't remember, I don't see them on this commit change-list

Reply:
Whoops. I forgot we even had those. Removed now!